### DISCUSSION TITLE
Turning around underwater is clumsy in multiplayer

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15457] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

https://github.com/user-attachments/assets/37b3eef1-84da-4e02-a328-a8274e901cde

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.7.7.0 (Winter Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by A2ur3** [comment]
the issue seems has potentially been fixed in latest unstable

**Comment by Regalis11** [comment]
Yeah, this has been addressed in the unstable build, and the fix will be included in the next update. Sorry for the inconvenience!
